pkg/volume/plugins.go

```diff
@@ -367,6 +367,9 @@ type VolumeHost interface {
 
 	// Returns the event recorder of kubelet.
 	GetEventRecorder() record.EventRecorder
+
+	// SetKubeletError sets the ready state on the kubelet.
+	SetKubeletError(err error)
```

Comment:
can't this already be accomplished using GetKubeClient() and GetNodeName()? if so, let's make a utility method that does the setting rather than adding to this interface and making unrelated objects implement empty functions

Reply:
I'm not familiar with the method you're referring to here. How do I set Kubelet ready status using KubeClient and NodeName, you mean by updating the API object directly? will Kubelet not then overwrite the ready status next time it polls the readystatusfuncs?

Reply:
yeah, read further and figured out what this is for. made a suggestion earlier about a kubelet-specific volume host interface